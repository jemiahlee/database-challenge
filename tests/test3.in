set var this_is_a_string
get var
END
