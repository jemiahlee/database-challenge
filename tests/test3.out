this_is_a_string
